spec/manifest_pr_checker_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module Danger
+  describe Danger::ManifestPRChecker do
+    it 'should be a plugin' do
+      expect(Danger::ManifestPRChecker.new(nil)).to be_a Danger::Plugin
+    end
+
+    describe 'with Dangerfile' do
+      before do
+        @dangerfile = testing_dangerfile
+        @my_plugin = @dangerfile.manifest_pr_checker
+      end
+
+      it 'returns an error when a PR changed the Gemfile but not Gemfile.lock' do
```

Comment:
Nitpick

```suggestion
      it 'returns an error when a PR changed the Gemfile but not the Gemfile.lock' do
```

or:

```suggestion
      it 'returns an error when a PR changed Gemfile but not Gemfile.lock' do
```

~~(_I prefer the first suggestion_)~~ Update: I noticed the other example doesn't use an article for either `Gemfile[.lock]`, so maybe the second is more consistent.

Reply:
I think I'll just change everything to use an article, to make it consistent. It also sounds better to me 👍